frontend/src/lib/apiClient.js

```diff
@@ -102,6 +102,19 @@ const createRecipe = recipe => apiClient.post('api/v1/recipes', {
   },
 });
 
+// COMMENTS
+
+const getComments = id => apiClient.get(`api/v1/comments/${id}`);
```

Comment:
I think you probably want this to take a `recipeID` and call `api/v1/recipes/${recipeID}/comments`. This presently gets a single comment.